core/src/main/java/org/bitcoinj/core/Peer.java

```diff
@@ -870,6 +869,18 @@ protected void processTransaction(final Transaction tx) throws VerificationExcep
         return resultFuture;
     }
 
+    /**
+     * Build a GetDataMessage to query multiple transactions by ID
+     * @param txIds A set of transaction IDs to query
+     * @return A GetDataMessage that will query those IDs
+     */
+    private GetDataMessage buildMultiTransactionDataMessage(Set<Sha256Hash> txIds) {
+        GetDataMessage getdata = new GetDataMessage(params);
+        txIds.forEach(txId ->
+            getdata.addTransaction(txId, vPeerVersionMessage.isWitnessSupported()));
+            return getdata;
```

Comment:
Is this indentation correct? Seems like it should be one indent less.

Reply:
Fixed and force-pushed.